tasks/modprobeib.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: disable ipoib_enhanced
+  ansible.builtin.lineinfile:
+    path: /etc/modprobe.d/ib_ipoib.conf
+    insertafter: '^alias'
+    line: options ib_ipoib ipoib_enhanced=0
```

Comment:
There should be a tunable to enable or disable `ipoib_enhanced`. Some users of this role would need IPoIB enhanced (or at least would like to stick to the default values).

Reply:
oui je vais ajouter une condition boolen ipoib_enhanced=true dans les variable

Reply:
`infiniband_ipoib_enhanced: True` added to main file